src/coreclr/System.Private.CoreLib/src/System/GC.CoreCLR.cs

```diff
@@ -865,7 +865,9 @@ private static unsafe void ConfigCallback(void* configurationContext, void* name
                 Configurations = new Dictionary<string, object>()
             };
 
-            _EnumerateConfigurationValues(Unsafe.AsPointer(ref context), &ConfigCallback);
+#pragma warning disable CS8500
```

Comment:
💡 As an aid to the reader, can these be updated to include the title for CS8500?

Reply:
My request would only be for the ones in this PR. I'm not a maintainer on this repository, so I can't speak to the preferred comment if there is more than one form seen in other code.